server/src/main/java/org/jetbrains/bsp/bazel/server/sync/BazelPathsResolver.kt

```diff
@@ -50,15 +50,55 @@ class BazelPathsResolver(private val bazelInfo: BazelInfo) {
         fileLocation.isSource && !fileLocation.isExternal
 
     fun labelToDirectoryUri(label: Label): URI {
-        val relativePath = extractRelativePath(label.value)
-        return resolveUri(bazelInfo.workspaceRoot.resolve(relativePath))
+        val absolutePath = if (isRelativeWorkspacePath(label.value)) {
```

Comment:
can u reuse `pathToDirectoryUri` here somehow?

Reply:
@allvpv you are the path master. Please take a look

Reply:
I managed to reuse it